### PR TITLE
fix Global Search/Tagging: regenerated services with latest API defin…

### PR DESCRIPTION
**Pull request by fabrizio-leoni**
…ition

## PR summary

Documentation updates.

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by fabrizio-leoni** [comment]
image

**Comment by fabrizio-leoni** [comment]
image

File: modules/global-search/src/main/java/com/ibm/cloud/platform_services/global_search/v2/GlobalSearch.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  */
 
 /*
- * IBM OpenAPI SDK Code Generator Version: 3.79.0-2eb6af3d-20230905-174838
+ * IBM OpenAPI SDK Code Generator Version: 3.82.1-2082d402-20231115-195014
```
Comment (padamstx) [inline modules/global-search/src/main/java/com/ibm/cloud/platform_services/global_search/v2/GlobalSearch.java:15]: Does is seem reasonable that your generated SDK code did not change other than the version of the generator???
If this is correct, then there's probably no reason to deliver this change.

Comment (padamstx) [reply modules/global-search/src/main/java/com/ibm/cloud/platform_services/global_search/v2/GlobalSearch.java:15]: Make sure you are following the instructions in https://github.com/IBM/platform-services-java-sdk/blob/main/update_service.md

**Comment by fabrizio-leoni** [comment]
I close this PR for wrong file changed.

**Pull request closed without merging**